python/nav/mibs/coriant_groove_mib.py

```diff
@@ -207,3 +207,25 @@ def _get_sensor_description(self, column):
         mib_object = self.nodes[column].raw_mib_data
         description = mib_object.get("description", "").strip()
         return description.split("\n")[0]
+
+
+def _fix_text_columns():
+    """Updates CoriantGrooveMib.text_columns to include all known text columns.
+
+    The MIB modules itself marks most of these up as OctetString, rather than
+    DisplayString, causing the SNMP library to return them all as bytes objects
+    rather than strings.
```

Comment:
I'm assuming you're doing it this way since the mib-file itself is larger than 30K lines? If not, a comment may be wise.

Reply:
@hmpf: No. The MIB module is autogenerated from Coriant's published MIB module (ASN.1 syntax), using the `smidump` command line tool. We don't want to touch the generated output, as it may be re-generated at any time.

Reply:
Add that as a comment!